admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -48,10 +50,31 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 				.not( '.toggle-group' )
 				.on( 'click swipe contextmenu', self._setMasterToggles );
 
+			$bgBox
+				.find( '.table-help td p' )
+				.attr( 'style', 'height: 0em; opacity: 0%; position: relative; z-index:-1' );
 			$bgBox.find( '.dashicons-editor-help' ).on( 'click', self._toggleHelp );
```

Comment:
```JS
$bgBox.find( '.dashicons-editor-help' ).on( 'click', self._toggleHelp );
$bgBox.find( '.help-icon' ).on( 'click', self._toggleHelp );
```

Both of these lines are listening for a click and performing the same action. Is it possible to combine them into one line?

Reply:
fixed